rust/stackable-cockpit/src/engine/minikube/mod.rs

```diff
@@ -45,8 +50,8 @@ impl MinikubeCluster {
         info!("Creating local cluster using minikube");
 
         // Check if required binaries are present
-        if !binaries_present(&["docker", "minikube"]) {
-            return Err(MinikubeClusterError::MissingDepsError);
+        if let Some(binary) = binaries_present_with_name(&["docker", "minikube"]) {
```

Comment:
Is this worth spinning out into some kind of preconditions module?

Reply:
Yes might be. I guess it would also include other precondition checks, like cluster resources. Should this be included in this PR or do we add it to a separate one?

Reply:
Probably fine to wait, more of a note for the future.

Reply:
Alright, added it to my TODO list.